Sources/Datadog/DatadogInternal/Upload/URLRequestBuilder.swift

```diff
@@ -6,16 +6,18 @@
 
 import Foundation
 
+internal typealias URLRequestBuilder = DDURLRequestBuilder
+
 /// Builds `URLRequest` for sending data to Datadog.
-/* public */ internal struct URLRequestBuilder {
-    enum QueryItem {
+public struct DDURLRequestBuilder {
+    public enum QueryItem {
```

Comment:
I'm doing the trick with `typealias` here to avoid renaming surgery in core. The SDK still uses `URLRequestBuilder` name, but in public API it's `DDURLRequestBuilder` to avoid ambiguity.

Reply:
Even tho this is marked public, it will be exposed by `DatadogInternal` which shouldn't be imported by users. Importing this module results in using private APIs, so we don't have to care much about ambiguity.

Reply:
Indeed 👍. IMO sounds good to revert this rename once we have `DatadogInternal` module separated (not the case right now, as we will be exposing `DDURLRequestBuilder` in SDK's public API).

Reply:
Agreed 👍